### ISSUE TITLE
using 64 grids

### ISSUE THREAD

**Issue by ergodicbreak** [schollz/paracosms#16]
Hello, I've been using paracosms since release and it's great, thanks for an amazing script. 

Up to now I haven't used the grid much but I'm starting to get into that more lately, however I have a 64 grid. It's workable but I was thinking of having a go at improving the experience a bit, if you don't mind I have some questions. 

It seems all the grid functionality (other than loading the grid in `lib/paracosms.lua`) is in the `ggrid` file, is that correct? 

My plan would be to reduce the available sample slots on the grid to 56 (you could still access sample slots greater than 56 on the norns), clamp any loading of samples so there's only 8 per row, and decrease the time scale of sample start/end/duration. Assuming I get the math right in the relevant functions (famous last words), would there be any other gotchas with a plan like that?

Thanks!

**Comment by schollz** [comment]
yeah! that sounds great. it should just work I believe, the shading of the buttons will be weird of course but I would definitely love to have a PR if you want to make one!

**Comment by ergodicbreak** [comment]
Most of the changes have looked straightforward enough, though there are a few changes I'm not sure about, for example 

`      params:set(datti.."sampleStart",util.round(util.linlin(1,64,0,1,id),1/64)) -- TODO assumes width of 16?`

_https://github.com/ergodicbreak/paracosms/blob/52069eccf9157d892d425e12ffbd9754f2c08bf6/lib/ggrid.lua#L92C7-L92C99_

Is it safe to leave this line as is though my beat resolution on the 64 grid is half beats, rather than quarters like on the 128 grid (I may be totally misunderstanding this!)?

There are a couple of other things, like midigrid 64 support and possibly supporting switching grids on the fly that I need to think about some more, all those are still TODO in the ggrid file. 

Thanks for making your script so accessible!

**Comment by schollz** [comment]
yeah that should be safe!
